js-structure_spacerocket/js/launchSequence.js

```diff
@@ -1 +1,20 @@
 // Implement the launch sequence function here and export it as the default export.
+import { NFSAT } from "./payload/satellites.js";
+import { FISHSAT } from "./payload/satellites.js";
+import { fuel } from "./core/fuel.js";
+import { countdown } from "./core/countdown.js";
+import { liftoff } from "./core/liftoff.js";
+import { deployPayload } from "./core/deploy.js";
+
+export function launchFunction() {
```

Comment:
The <code>launch()</code> function must be the default export of the module.
The function should be call `lunch`